ocaml/idl/datamodel_pool.ml

```diff
@@ -1322,6 +1322,10 @@ let t =
             ~default_value:(Some (VRef null_ref)) "repository_proxy_password"
             "Password for the authentication of the proxy used in syncing with \
              the enabled repositories"
+        ; field ~lifecycle:[] ~ty:Bool ~default_value:(Some (VBool false))
```

Comment:
You should add `~qualifier:DynamicRO`. I'm not sure why the compiler didn't complain about this.

Reply:
Thanks, although it should be RW right?